code/drasil-gool/lib/GOOL/Drasil/ClassInterface.hs

```diff
@@ -0,0 +1,518 @@
+{-# LANGUAGE TypeFamilies #-}
+
+module GOOL.Drasil.ClassInterface (
```

Comment:
I want @B-rando1 to also check this. I wonder if this is somehow an artifact of merge order, i.e. this file has moved and shouldn't be re-added anew.

Reply:
That would be my guess.  This file, as well as the folders it's in (`GOOL/Drasil` was switched to `Drasil/GOOL`), are from the older GOOL structure and should be removed.  It looks like it isn't referenced anywhere, so hopefully that's an easy step.

Reply:
I have removed the file and folder